index.html

```diff
@@ -340,3 +340,22 @@ <h3 id="render" data-export="" data-dfn-type="dfn">
 and will create user agents to represent those preferences on the web user's behalf.
   </p>
 </section>
+
+<section>
+  <h3 id="centralization" data-export="" data-dfn-type="dfn">
+The web is decentralized
+  </h3>
+  <p>
+    The web is a platform designed to be used and operated by as wide
+    a range of individuals and orgaizations as possible. 
+    We will not create web technologies that encourage centralization,
+    by favoring large or powerful organizations, to the detriment of smaller
+    organizations. We include in this concern both technologies
+    that explicitly promote centralization (e.g., systems that rely on
+    a single or small number of "trusted parties" to collect or pre-process data),
+    and technologies that would promote centralization in more subtle ways (e.g.,
+    systems that are prohibitively expensive to run, or systems that provide more
+    utility as the number of participants increase).
```

Comment:
This seems to rule out a large number of potentially useful things; effectiely, you're saying that the Web should _not_ be more useful/efficient at scale.

Reply:
Thats not what i mean. I'd be grateful for help rewording, or refining if the principal appeals.

My goal is to capture that it'd be bad to standardize systems that:

1. that require (either explicitly or in principal) user-bases bigger than what all but the largest system-participants / implementors can achieve. Examples could be K-anon systems with large K, or moderate K over a wide and uniform-ish distributed set of values, or noise-injection systems that require many users to recover a useable signal, etc.  
2. are expensive enough that they'd prohibit many/most system participants from implementing/running

Those goals seem distinguishable from systems that become marginally cheaper, or slightly more useful as usage increases.  Though i appreciate the need to for careful language to separate the two.

Reply:
Perhaps an example would help: is there a specification you have in mind where a design decision was able to discriminate along these lines? Or one that failed to, when it could have?